src/components/donate/Sepa.js

```diff
@@ -215,6 +216,40 @@ export default function Register(props) {
               )}
             />
           </Grid>
```

Comment:
Please don't introduce yet another way of doing things, check src/components/Register.js and use the same, pretty please